packages/shared/src/tableDefinitions/mapObservationsTable.ts

```diff
@@ -68,7 +68,7 @@ export const observationsTableUiColumnDefinitions: UiColumnMapping[] = [
     uiTableId: "timeToFirstToken",
     clickhouseTableName: "observations",
     clickhouseSelect:
-      "if(isNull(completion_start_time), NULL,  date_diff('milliseconds', start_time, completion_start_time))",
+      "if(isNull(completion_start_time), NULL,  date_diff('seconds', start_time, completion_start_time))",
```

Comment:
this will not work in UI if we use decimals. Change to milliseconds and convert user entry to ms

Reply:
are seconds fine here? From my experience, we need milliseconds.

Reply:
[Fixed it in this commit](https://github.com/langfuse/langfuse/pull/4536/commits/36b5a4741d62da6a6f3cca11962598b048e9715e) to also allow both sorting and filtering by decimal second value :)